### DISCUSSION TITLE
[Sugestion] Radiator blocks

### DISCUSSION THREAD

**Discussion by MrSquid1** [tsunamayo/Starship-EVO#4404] [Suggestions]
Have a radiator block that disperses a lot of heat but is very fragile, maybe try to encourage it to be exposed for balance.

This would balance combat ships since without them the heat would start to build up making them weak points making them useful in civilian ships.

**Comment by commche** [comment]
I like this idea. this would counter my gripe about ships in general being slower than I like. The radiator could be like a grill, weak and disperse lots of heat when open, and stronger with less dispersion when closed

**Comment by tsunamayo** [comment]
Hi, yep it has been suggested many time, but I want to go further down balancing combat before doing this. Thanks
